Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift

```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
```

Comment:
remark(no-blocking): wrong name of file

Reply:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689